config/server1.properties

```diff
@@ -10,4 +10,4 @@
 # CONDITIONS OF ANY KIND, either express or implied.
 #
 host.name=localhost
-port=6667
+port=6670
```

Comment:
If you don't mind, can you also create 3 more files for ssl. (serverSsl1.properties, serverSsl2.properties, etc) for ssl purposes